scripts/upgrade

```diff
@@ -103,6 +103,16 @@ ynh_system_user_create "$app"
 # Create a dedicated php-fpm config
 ynh_add_fpm_config
 
+#=================================================
+# SPECIFIC UPGRADE
+#=================================================
+# UPDATE PLUGINS
+#=================================================
+
+pushd "$final_path"
+	exec_as "$app" "bin/gpm update -y"
+popd
```

Comment:
I'm not found of this, pushd/popd, it works really fine, but I find it difficult to use for newcomers. Using a sub-shell and having a `cd` gives the same result and is easier to read (for newcomers).

Reply:
No problem :)

```sh
exec_as "$app" "cd "$final_path" && bin/gpm update -y"
```
This should be the alternative.